src/test.rs

```diff
@@ -148,15 +152,10 @@ fn it_works_decode_with_method_cooldown() {
         95, 116, 101, 115, 116, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 222,
         0, 0, 0,
     ];
-    let c: CENNZnutV0 = Decode::decode(&mut &encoded[..]).expect("It works");
+    let c: CENNZnut = Decode::decode(&mut &encoded[..]).expect("It works");
+    let c0 = CENNZnutV0::try_from(c).unwrap();
     assert_eq!(
-        c.get_module("module_test")
-            .expect("module exists")
-            .block_cooldown,
-        Some(86_400)
-    );
```

Comment:
Can we try to keep this `block_cooldown` assertion?

Reply:
Yes. I guess the removal was just a mistake.